x/axelarnet/message_handler.go

```diff
@@ -173,7 +173,7 @@ func handleMessage(ctx sdk.Context, n types.Nexus, sourceAddress nexus.CrossChai
 		sourceAddress,
 		recipient,
 		crypto.Keccak256Hash(msg.Payload).Bytes(),
-		nexus.Sent,
+		nexus.Approved,
```

Comment:
Is this change just so that the `ExecuteMessage` can then pick it up?

Reply:
Yes, and so it's not automatically picked up the evm end blocker. The end blocker iterates through messages with the `Sent` status.